package test.java;
import java.io.IOException;
import java.util.ArrayList;

public class DataDrivenTestDemo {

    public static void main(String[] args) throws IOException {

        DataDrivenMethod dd = new DataDrivenMethod();
        /*ArrayList data = dd.getData("Password_2");

        System.out.println(data.get(1));
        System.out.println(data.get(2));
        System.out.println(data.get(3));
        System.out.println(data.get(4));
        System.out.println(data.get(5));*/
    }
}
